src/Essentials/src/Types/Shared/WebUtils.shared.cs

```diff
@@ -1,45 +1,50 @@
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Text;
 
 namespace Microsoft.Maui.ApplicationModel
 {
 	static class WebUtils
 	{
 		internal static IDictionary<string, string> ParseQueryString(string url)
```

Comment:
Is there some standard `System.Uri` parsing code we can call instead of writing our own?

It seems like there should be a BCL way to return a dictionary of key/value paris.

Reply:
It does seem like there should be a class for this in the BCL but afaik, there isn't.

It is definitely not part of Uri.

Reply:
This is the new one:

https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.webutilities.queryhelpers.parsequery?view=aspnetcore-7.0#microsoft-aspnetcore-webutilities-queryhelpers-parsequery(system-string)

Can we bring in the source for it and make it `internal`?

https://source.dot.net/#Microsoft.AspNetCore.WebUtilities/QueryHelpers.cs,1c1b023fbf834a3d